### DISCUSSION TITLE
[ERROR][yaml scanner] Default configuration prod.yaml

### DISCUSSION THREAD

**Discussion by Mat-moran** [Tecnativa/doodba#362] [Q&A] [answered]
Hi, I am starting with doodba (BEAUTIFUL project, congrats for the work to all the team!!) 
I am trying to configure the simplest (default values from copier template) doodba production enviroment with the [global inverse proxy](faq.md#how-to-bootstrap-the-global-inverse-proxy) default configuration. I have a domain pointing to the VPS and I use it during the copier initial configuration steps.

### 1 ERROR

When I run the command
`docker-compose -f prod.yaml up -d`

I get the next error:

> ERROR: yaml.scanner.ScannerError: mapping values are not allowed here in "./prod.yaml", line 27, column 41`

This is because of the double points at the end of all the lines like the next one:
`traefik.main-0.frontend.rule: Host:`

Is there is a bug in the creation of prod.yaml from copier or am I doing something wrong??

### 2 ERROR (after rewriting the lines of the first error)
When I delete the double points at the end of each line like the previous one I get the next error:
> ERROR: Network "prueba_default" needs to be recreated - option "encrypted" has changed

Any suggestions?? Am I doing something wrong?

Thanks and regards

**Comment by joao-p-marques** [comment, marked as answer]
Hi @Mat-moran 

Thank you for the kind words and for reporting your issue! :smiley: 

Can you specify what you have defined in the `domains_prod` variable? You can see it in the `copier-answers.yml` file.
A more detailed specification of the contents of `prod.yaml` would also be of some help.
You can anonymize some details if you prefer.

Also, what version of traefik are you using?

**Reply by Mat-moran** [reply]
Thanks to you @joao-p-marques 

`domains_prod: www.pruebas2.dminternational.es`
`domain_test: test.dminternational.es`

**Reply by joao-p-marques** [reply]
Thank you! Is that the only content of those variables?

As explained in the help text for the interactive questions, you should provide a structure similar to what you see in https://github.com/Tecnativa/doodba-copier-template/blob/stable/copier.yml#L198

Take a look at the `placeholder`, for example. You should, at least, provide a yaml list with the key `hosts`, which itself will be a list of domains to use in the production context.

Hope this is of some help :smiley:

**Reply by Mat-moran** [reply]
OK my fault sorry!! I will check it!!
Thanks a lot ;-)
